requirements-dev.txt

```diff
@@ -55,27 +55,28 @@ pytest==7.2.1
 pytz==2022.7.1
 pyyaml==6.0 ; python_version >= '3.6'
 requests==2.28.2 ; python_version >= '3.7' and python_version < '4'
-setuptools==67.1.0 ; python_version >= '3.7'
+setuptools==67.3.2 ; python_version >= '3.7'
 six==1.16.0 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 smmap==5.0.0 ; python_version >= '3.6'
 snowballstemmer==2.2.0
 sphinx==5.3.0
-sphinx-rtd-theme==1.1.1
+sphinx-rtd-theme==1.2.0
 sphinxcontrib-applehelp==1.0.4
 sphinxcontrib-devhelp==1.0.2
 sphinxcontrib-htmlhelp==2.0.1
 sphinxcontrib-httpdomain==1.8.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+sphinxcontrib-jquery==2.0.0 ; python_version >= '3.1'
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-openapi==0.7.0
 sphinxcontrib-plantuml==0.24.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
-stevedore==4.1.1 ; python_version >= '3.8'
+stevedore==5.0.0 ; python_version >= '3.8'
```

Comment:
Do we load code dynamically using this library or what do we do with it?

Reply:
Some of our dependencies dynamically use it.

Our dependencies are maintained in the Pipfile. The `requirements-*.txt` is an export from the `Pipfile.lock`, guaranteeing that we are also looking at and pinning the dynamic dependencies. 

https://github.com/vmware/repository-service-tuf-api/blob/0a2d7dea2d6a4503761982faf61ec975ca886fe4/Pipfile#L6-L30